From 6775053ab1aae581b212f40cbf3926c3e64d8ab0 Mon Sep 17 00:00:00 2001
From: leigh123linux <leigh123linux@googlemail.com>
Date: Sun, 5 Feb 2017 12:47:20 +0000
Subject: [PATCH] fix missing libmiam-acoustid.so on linux

---
 src/acoustid/acoustid.pro | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/acoustid/acoustid.pro b/src/acoustid/acoustid.pro
index b3c972d..c63041a 100644
--- a/src/acoustid/acoustid.pro
+++ b/src/acoustid/acoustid.pro
@@ -49,6 +49,8 @@ TARGET      = miam-acoustid
 
 !macx {
     LIBS += -L$$OUT_PWD/../core/ -lmiam-core -ltag -lchromaprint
+    target.path = /usr/lib$$LIB_SUFFIX/
+    INSTALLS += target
 }
 macx {
     QMAKE_SONAME_PREFIX = @executable_path/../Frameworks
